### PR TITLE
split-pr-part-two - add 5 files

### PR DESCRIPTION
**Pull request by michaelkernaghan**
split up the big Pr

### CODE REVIEW

File: integration-tests/config.ts [hunk 1]
```diff
@@ -101,18 +101,18 @@
   signerConfig: {
     type: SignerType.EPHEMERAL_KEY as SignerType.EPHEMERAL_KEY,
     keyUrl: 'http://key-gen-1.i.tez.ie:3010/mondaynet',
-    requestHeaders: { Authorization: 'Bearer taquito-example' },
+    requestHeaders: { Authorization: 'Bearer taquito' },
```
[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/config.ts:104]: we are ok with this

[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/config.ts]: the file needs to be updated to use the example bearer token

[the remark this answers is not in the document]
Comment (michaelkernaghan) [reply integration-tests/config.ts]: it was updated.

**Review by zainen — APPROVED** [review]
LGTM!

**Review by roxaneletourneau — COMMENTED** [review]
All the changes in this PR seem to also appear in the following PR: https://github.com/ecadlabs/taquito/pull/1811/files
The current Pr can be closed without merging.

**Comment by michaelkernaghan** [comment]
closed as duplicate

**Pull request closed without merging**